### DISCUSSION TITLE
[7일차-지원] webpack에 대해 설명해보세요

### DISCUSSION THREAD

**Discussion by smarfy99** [zzangCS/zzangInterview#27] [프론트 면접 질문]
- polyfill
- bundle
중점으로

**Comment by ez1n** [comment]
https://devowen.com/288

**Comment by smarfy99** [comment]
여러 파일을 하나 이상의 파일로 합쳐주는 JS 번들러

**Comment by skyhanull** [comment]
웹팩은 모듈을 번들링 해주는 모듈 번들러입니다
여기서 모듈 번들링이란 웹 애플리케이션을 구성하는 몇십, 몇백 개의 자원들을 하나의 파일로 병합 및 압축 해주는 동작을 모듈 번들링이라고 합니다.
